Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.h

```diff
@@ -395,10 +391,15 @@ using GstStateLocker = ExternalLocker<void, gstStateLock, gstStateUnlock>;
 template <typename T>
 class GstIteratorAdaptor {
 public:
-    GstIteratorAdaptor(GUniquePtr<GstIterator>&& iter)
-        : m_iter(WTFMove(iter))
+    GstIteratorAdaptor(GstIterator* /* transfer full */ iter)
```

Comment:
IMHO, I would prefer to keep the GUniquePtr&& here. Besides, this would be a fly-by, out of the scope of the patch.

Reply:
It is definitely in scope of the patch. Quoting the changelog:

Several inline functions had to move to the cpp unit and the IteratorAdaptor dependency on
GUniquePtr was removed, so that GUniquePtrGStreamer no longer needs to be included in
GStreamerCommon.h.